### PR TITLE
<CardGalleryItem /> - added prop `disabled` for `primaryAction`

### PR DESCRIPTION
**Pull request by Barackos**
### 🔦 Summary
Following our slack discussion
https://wix.slack.com/archives/C39FTGUBZ/p1567515579153500

Feature addition:
`primaryAction` will have the option to be disabled.

In order to avoid harm to other projects, the `disabled` prop is `false` by default.

### ✅ Checklist
- [✅] 👨‍💻 API change is approved by the UI Infra Developers 
@shlomitc 

- [✅] 👨‍🎨 UX change is approved by the Design System UX 
@wuwa 

- 📚 Change is documented
  - [✅] Story
  - [✅] API description
  - [ ] Other (explain)
- 🔬 Change is tested
  - [✅] Component
  - [✅] Visual test

### CODE REVIEW

File: src/CardGalleryItem/CardGalleryItem.js [hunk 1]
```diff
@@ -38,6 +39,10 @@ class CardGalleryItem extends React.Component {
       label: PropTypes.node,
       /** On click handler of primary action button and of the whole card */
       onClick: PropTypes.func,
+      /** Option to disable the button */
```
Comment (mosheKerbel) [inline src/CardGalleryItem/CardGalleryItem.js]:
```suggestion
      /** Disable the primary action button */
```

File: src/CardGalleryItem/CardGalleryItem.js [hunk 2]
```diff
@@ -38,6 +39,10 @@ class CardGalleryItem extends React.Component {
...
+      disabled: PropTypes.bool,
+      /** Option to add tooltip message for button, when disabled */
```
Comment (mosheKerbel) [inline src/CardGalleryItem/CardGalleryItem.js]:
```suggestion
      /** Message to be displayed when primary action button is disabled */
```

File: src/CardGalleryItem/CardGalleryItem.uni.driver.js [hunk 3]
```diff
@@ -40,6 +40,8 @@ const cardGalleryItemDriverFactory = base => {
       await (await getPrimaryAction(base)).click();
     },
     getPrimaryActionLabel: async () => (await getPrimaryAction(base)).text(),
+    getPrimaryActionDisabledState: async () =>
```
Comment (mosheKerbel) [inline src/CardGalleryItem/CardGalleryItem.uni.driver.js]:
```suggestion
    isPrimaryActionDisabled: async () =>
```

**Comment by LittleTal** [comment]
@Barackos  Looks good:)

**Comment by mosheKerbel** [comment]
failed due to flaky unrleated tests.

**Pull request merged**